client/src/app/components/nova/landing/LandingEpochSection.tsx

```diff
@@ -91,21 +103,27 @@ const LandingEpochSection: React.FC = () => {
             </div>
 
             <div className="epoch-section__controls">
-                <button className="icon-button">
-                    <span className="epoch-section__previous">
-                        <ArrowUp width={20} height={20} />
-                    </span>
-                </button>
+                <a href={epochIndex > 0 ? `/${network}/epoch/${epochIndex - 1}` : undefined} target="_blank" rel="noopener noreferrer">
```

Comment:
We should not open in new tab

Reply:
I think we have to use `import { Link } from "react-router-dom";` like the rest of the explorer to better route instead of reloading the whole window like it does now